File: proofs/planetary_song.js
// JavaScript proof
const BASE_FREQ = 432;
const SHARDS = 71;

const frequencies = Array.from({length: SHARDS}, (_, i) => 
  BASE_FREQ + (BASE_FREQ * (i + 1) / SHARDS)
);

const [min, max] = [frequencies[0], frequencies[SHARDS-1]];
const ratio = max / min;

console.log(`Min: ${min.toFixed(2)} Hz, Max: ${max.toFixed(2)} Hz, Ratio: ${ratio.toFixed(4)}`);
console.assert(Math.abs(ratio - 2.0) < 0.05, `Not an octave: ${ratio}`);
console.log(`✓ JavaScript: Octave verified [${min.toFixed(2)}-${max.toFixed(2)} Hz]`);

File: proofs/planetary_song.rs
// Rust proof
fn main() {
    const BASE_FREQ: f64 = 432.0;
    const SHARDS: usize = 71;
    
    let frequencies: Vec<f64> = (0..SHARDS)
        .map(|i| BASE_FREQ * (i as f64 + 1.0) / SHARDS as f64)
        .collect();
    
    let min = frequencies.first().unwrap();
    let max = frequencies.last().unwrap();
    let ratio = max / min;
    
    assert!((ratio - 2.0).abs() < 0.01, "Not an octave");
    println!("✓ Rust: Octave verified [{:.2}-{:.2} Hz], ratio: {:.4}", min, max, ratio);
}
